### PR TITLE
Update server.properties

### PR DESCRIPTION
**Pull request by NatsukiORG**

### CODE REVIEW

**Comment by evallll** [comment]
May I ask what is the purpose or reason behind disabling command blocks? As far as I am concerned, it is an integral part of kaboom to be able to use command blocks, this wouldn't do very good for the anarchy side of things and would break a lot of player bots which rely on command blocks.

**Comment by IuCC123** [comment]
I don't think this is a great idea, this ruins the point of Kaboom.

**Comment by vanquishes** [comment]
😂😂😂😂

**Comment by B2RT0SZM77** [comment]
bad idea

**Comment by oie1** [comment]
bad

**Comment by Parker2991** [comment]
the hell is the point in this lmao

**Review by Dinhero21 — COMMENTED** [review]
no

File: server.properties [hunk 1]
```diff
@@ -2,7 +2,7 @@
 enable-jmx-monitoring=false
 level-seed=
 rcon.port=25575
-enable-command-block=true
+enable-command-block=false
```
Comment (OptimisticDeving) [inline server.properties:5]: Revert this change :pray:

**Review by OptimisticDeving — CHANGES_REQUESTED** [review]

**Pull request closed without merging**